marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mergeNotification.cy.tsx

```diff
@@ -93,8 +101,11 @@ describe("Merge Notification Functionality From Explore Card View", () => {
       toolbar.closeNotificationModal();
     });
   });
-  it("Run Match and Merge steps to generate Notification Docs", () => {
+
+  it.only("Run Match and Merge steps to generate Notification Docs", () => {
+    cy.intercept("GET", "/api/flows/personJSON/latestJobInfo").as("personJSON");
     cy.waitUntil(() => toolbar.getRunToolbarIcon()).click();
+    cy.wait("@personJSON").wait("@personJSON");
```

Comment:
I suggest changing toogleExpandFlow to toggleFlowAccordion. The accordion is closed so you cannot runStep on line 112.
Also remember to remove the only in the it.

Reply:
Done!